src/ontology/mondo-edit.obo

```diff
@@ -461123,7 +461120,7 @@ synonym: "pituitary-dependent Cushing's disease" EXACT [doi:10.1093/jama/9780195
 xref: DOID:3946 {source="EFO:1001110", source="MONDO:equivalentTo"}
 xref: EFO:1001110 {source="MONDO:equivalentTo", source="MONDO:EFO"}
 xref: GARD:19699 {source="MONDO:GARD"}
-xref: ICD10CM:E24.0 {source="EFO:1001110", source="MONDO:equivalentTo", source="Orphanet:99892/ntbt", source="DOID:3946", source="Orphanet:99892", source="MONDO:directSiblingOf"}
+xref: ICD10CM:E24.0 {source="EFO:1001110", source="MONDO:equivalentTo", source="Orphanet:99892/ntbt", source="DOID:3946", source="Orphanet:99892"}
```

Comment:
removed incorrect `source="MONDO:directSiblingOf`, the `source="MONDO:equivalentTo"` remains